docs/cicd.md

```diff
@@ -29,13 +75,19 @@ Lanes for Android live in [./android/fastlane](./android/fastlane) and for iOS i
+| BROWSERSTACK_USERNAME   | Username for BrowserStack      | Password Manager        | [Appium REST API](https://www.browserstack.com/app-automate/rest-api)                      |
+| DELIVERINO_PRIVATE_KEY  | Base64 encoded PEM private key | Password Manager        | [Deliverino Settings](https://github.com/organizations/Integreat/settings/apps/deliverino) | [Deliverino](https://github.com/apps/deliverino) |
 
 ### Android Variables
 
-| Variable                    | Description                                                                                                | Where do I get it from? | Example | Reference                                                                                              |
-| --------------------------- | ---------------------------------------------------------------------------------------------------------- | ----------------------- | ------- | ------------------------------------------------------------------------------------------------------ |
-| GOOGLE_SERVICE_ACCOUNT_JSON | JSON for authentication in the Google Play Console as Release Manager. This should expire after two years. | Password Manager        | {...}   | [Service Account Docu](https://cloud.google.com/iam/docs/creating-managing-service-account-keys?hl=de) |
+| Variable                    | Description                                                                                                | Where do I get it from?                                                  | Reference                                                                                              |
+| --------------------------- | ---------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------ |
+| GOOGLE_SERVICE_ACCOUNT_JSON | JSON for authentication in the Google Play Console as Release Manager. This should expire after two years. | Password Manager                                                         | [Service Account Docu](https://cloud.google.com/iam/docs/creating-managing-service-account-keys?hl=de) |
+| KEYSTORE_BASE64             | Base64 encoded encrypted keystore                                                                          | Password Manager                                                         | -                                                                                                      |
+| KEYSTORE_ENCRYPTION_KEY     | Password for decrypting the keystore using OpenSSL                                                         | Password Manager                                                         | -                                                                                                      |
```

Comment:
You are using OpenGPG right now. I think OpenSSL is the simpler tool to use for this. We should do it like in Integreat.

Reply:
In Integreat we are doing this with Fastlane in the keystore-lane. If we do not clone the credentials - as it is now - decrypt it in circleci directly is much shorter (just one step and one line comand).

Also i do not get, why you prefer OpenSSL. I read: "PGP/GPG has proven highly resistant to cryptographic attack and is a preeminent tool for secure communications. OpenSSL is more known for network security, but it also has tools useful for most aspects of encrypting flat files." So i guess GPG would be perfectly fine for this use case? But maybe I miss your point.

Anyway: I adjusted the docs and will add to the new keystore-handling-ticket, that the decryption can be done within fastlane, after cloning the credentials.

Reply:
OpenSSL is just simpler, both works fine though